CHANGELOG.txt

```diff
@@ -1,3 +1,10 @@
+1.2.2 (doi: )
```

Comment:
please use "master" for ongoing development changelog

Reply:
resolved